langgraph/src/tests/pregel.test.ts

```diff
@@ -877,6 +877,76 @@ describe("StateGraph", () => {
       },
     });
   });
+
+  // TODO: fix this test
+  xit("can invoke a nested graph", async () => {
```

Comment:
@nfcampos, I copied [this test](https://github.com/langchain-ai/langgraph/pull/345/files#diff-9f99beb90b66284c77ac257b4c0dfc06dcb998302f7c6740d84759460f704f53), but I haven't figured out where the fix needs to be in the existing implementation (which will probably be changed later). Ideally, I'd like to get this fixed/working before merging. I'll ask you for help in the morning.